bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx

```diff
@@ -31,13 +30,14 @@ export default class ActionsBar extends Component {
           <MuteAudioContainer />
           <JoinAudioOptionsContainer
             open={this.openJoinAudio.bind(this)}
-            close={() => {exitAudio();}}
+            close={() => {this.props.handleExitAudio();}}
```

Comment:
You dont need the arrow function. `close={this.props.handleExitAudio.bind(this)}` like the open function.
And probably `handleJoinAudio` would be a better name to the open function.